docs/_tutorials/rehearsal/rehearsal.rst

```diff
@@ -0,0 +1,70 @@
+Rehearsal
+---------
+
+A popular and efficient method in continual learning is to do **rehearsal**. Aka
+the action of reviewing a limited amount of previous images.
+
+Continuum provides a helper class to manage this memory and functions to
+samples ("to herd") examples into the memory.
+
+How to use the memory:
+
+.. code-block:: python
+
+    from torch.utils.data import DataLoader
+
+    from continuum import ClassIncremental
+    from continuum.datasets import CIFAR100
+    from continuum import rehearsal
+
+    scenario = ClassIncremental(
+        CIFAR100(data_path="my/data/path", download=True, train=True),
+        increment=10,
+        initial_increment=50
+    )
+
+    memory = rehearsal.RehearsalMemory(
+        memory_size=2000,
+        herding_method="barycenter"
```

Comment:
Do we want the herding methods for only RehearsalMemory or for both TaskSet and RehearsalMemory?
Maybe we can add a parameter function into "get_samples" method to define how data points are selected

Reply:
I feel it's better to be separated.